index.js

```diff
@@ -17,6 +20,19 @@ server.get("/api/:whichtable", async (req, res) => {
   }
 });
 
+server.post("/api/:whichtable", async (req, res) => {
+  let table = "projects";
+  if(req.params.whichtable ==="actions"){
```

Comment:
Loved how you use an `if` statement here to change to **actions** based on the `req.params` coming in. Keep up the logical thinking of these use cases! 👍